Cargo.toml

```diff
@@ -55,7 +55,7 @@ serde = "1.0.0"
 clippy = { version = "=0.0.162", optional = true }
 chrono = { version = "0.4.0", features = ["serde"] }
 comrak = { version = "0.1.9", default-features = false }
-ammonia = "0.7.0"
+ammonia = { git = "https://github.com/notriddle/ammonia" }
```

Comment:
Shouldn't this dependency be specified as `{ git = "https://github.com/notriddle/ammonia", rev = "46054ff" }` to ensure reproducible builds ?

Reply:
Good idea! I was thinking `Cargo.lock` storing the rev would be sufficient, but this is better.